foundation-extensions/experience-platform-extension/edge-network-api-reference.md

```diff
@@ -278,7 +344,269 @@ public class EdgeEventHandle {
 }
 ```
 
-Use this class when calling the sendEvent API with EdgeCallback.
+{% endtab %}
+
+{% tab title="iOS (AEP 3.x)" %}
+
+### Swift
+
+```swift
+@objc(AEPEdgeEventHandle)
+public class EdgeEventHandle: NSObject, Codable {
+    /// Payload type
+    @objc public let type: String?
+
+    /// Event payload values
+    @objc public let payload: [[String: Any]]?
+}
+```
+{% endtab %}
+{% endtabs %}
+
+Use this class when calling the [sendEvent](edge-network-api-reference.md#sendevent) API with EdgeCallback.
+
+### ExperienceEvent
+
+Experience Event is the event to be sent to Adobe Experience Platform Edge Network.
+The XDM data is required for any Experience Event being sent using the Edge extension.
+
+{% tabs %}
+{% tab title="Android" %}
+
+### Java
+
+```Java
+public final class ExperienceEvent {
```

Comment:
This ExperienceEvent class is missing the nested Builder static class. All the Builder APIs should be nested in the `public static class Builder` block. Also, there are two public ExperienceEvent APIs, `getData` and `getXDMSchema`, which are missing. 

https://git.corp.adobe.com/dms-mobile/platform-extension/blob/dev/code/edge/src/main/java/com/adobe/marketing/mobile/ExperienceEvent.java#L36

Reply:
Thanks.  Updated again.